service/src/main/java/bio/terra/landingzone/stairway/flight/CromwellStepsDefinitionProvider.java

```diff
@@ -6,22 +6,7 @@
 import bio.terra.landingzone.library.landingzones.definition.ResourceNameGenerator;
 import bio.terra.landingzone.library.landingzones.definition.factories.ParametersResolver;
 import bio.terra.landingzone.library.landingzones.definition.factories.validation.InputParametersValidationFactory;
-import bio.terra.landingzone.stairway.flight.create.resource.step.CreateAksStep;
-import bio.terra.landingzone.stairway.flight.create.resource.step.CreateAppInsightsStep;
-import bio.terra.landingzone.stairway.flight.create.resource.step.CreateBatchAccountStep;
-import bio.terra.landingzone.stairway.flight.create.resource.step.CreateBatchLogSettingsStep;
-import bio.terra.landingzone.stairway.flight.create.resource.step.CreateLogAnalyticsDataCollectionRulesStep;
-import bio.terra.landingzone.stairway.flight.create.resource.step.CreateLogAnalyticsWorkspaceStep;
-import bio.terra.landingzone.stairway.flight.create.resource.step.CreatePostgresLogSettingsStep;
-import bio.terra.landingzone.stairway.flight.create.resource.step.CreatePostgresqlDbStep;
-import bio.terra.landingzone.stairway.flight.create.resource.step.CreatePrivateEndpointStep;
-import bio.terra.landingzone.stairway.flight.create.resource.step.CreateRelayNamespaceStep;
-import bio.terra.landingzone.stairway.flight.create.resource.step.CreateStorageAccountCorsRules;
-import bio.terra.landingzone.stairway.flight.create.resource.step.CreateStorageAccountStep;
-import bio.terra.landingzone.stairway.flight.create.resource.step.CreateStorageAuditLogSettingsStep;
-import bio.terra.landingzone.stairway.flight.create.resource.step.CreateVnetStep;
-import bio.terra.landingzone.stairway.flight.create.resource.step.GetManagedResourceGroupInfo;
-import bio.terra.landingzone.stairway.flight.create.resource.step.ValidateLandingZoneParametersStep;
+import bio.terra.landingzone.stairway.flight.create.resource.step.*;
```

Comment:
I think we try to avoid * import.

Reply:
I think the formatter did this, at least I did not do this on purpose